addons/account_asset/models/account_invoice.py

```diff
@@ -12,6 +12,16 @@
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
+    @api.model
+    def _refund_cleanup_lines(self, lines):
+        result = super(AccountInvoice, self)._refund_cleanup_lines(lines)
+        for i in xrange(0, len(lines)):
```

Comment:
Does not work in Python 3, please use `range`. Or just use `enumerate(lines)` in the first place.